sw/device/tests/entropy_src_kat_test.c

```diff
@@ -88,10 +88,19 @@ static void flush_sha3_conditioner(dif_entropy_src_t *entropy_src) {
   CHECK_DIF_OK(dif_entropy_src_conditioner_start(entropy_src));
   stop_sha3_conditioner(entropy_src);
 
+  int fail_count = 0;
+  dif_result_t op_result;
+
   // Read (and discard) the resulting seed.
   uint32_t got[kEntropyFifoBufferSize];
   for (size_t i = 0; i < ARRAYSIZE(got); ++i) {
-    CHECK_DIF_OK(dif_entropy_src_non_blocking_read(entropy_src, &got[i]));
+    op_result = dif_entropy_src_non_blocking_read(entropy_src, &got[i]);
+    if (op_result == kDifUnavailable) {
+      fail_count++;
+      CHECK(fail_count < KAT_TEST_TIMEOUT_ATTEMPTS);
```

Comment:
s/`KAT_TEST_TIMEOUT_ATTEMPTS`/`kKatTestTimeoutAttempts`/ We recommend using enum constants instead to improve debuggability. See how `kEntropyFifoBufferSize` is defined in this test case.

Reply:
Done.